tests/org.dataflowanalysis.analysis.tests/src/org/dataflowanalysis/analysis/tests/dfd/MicroSecEndTest.java

```diff
@@ -238,39 +299,58 @@ private void hasLoggingServer(AbstractTransposeFlowGraph aTFG, Map<Integer, List
         }
     }
 
-    // For all microservices, there should exist a local logging agent decoupled from the microservice but deployed
-    // on the same host. Log data from microservices should not be send to the central logging system directly, but
-    // collected by the logging agent, written to a local file, and eventually send to the central system by it.
+    /**
+     * For all microservices, there should exist a local logging agent decoupled from the microservice but deployed
+     * on the same host. Log data from microservices should not be send to the central logging system directly, but
+     * collected by the logging agent, written to a local file, and eventually send to the central system by it.
+     * @param violationsSet
+     * @param variant
+     * @param node
+     */
     private void hasLocalLogging(Set<Integer> violationsSet, int variant, AbstractVertex<?> node) {
         if (hasNodeCharacteristic(node, "Stereotype", "internal") && !hasNodeCharacteristic(node, "Stereotype", "local_logging")) {
             violationsSet.add(10);
             violationsSet.add(11);
         }
     }
 
-    // The local logging agent should sanitize the log data and remove any PII, passwords, API keys, etc.
+    /**
+     *  The local logging agent should sanitize the log data and remove any PII, passwords, API keys, etc.
+     * @param violationsSet
```

Comment:
Just all small comment regarding the newly added javadoc: As you don't need the param-entries, you can just delete them instead of leaving them empty